packages/mobile/src/geth/networkConfig.ts

```diff
@@ -28,6 +28,8 @@ const pgpnpPubKeyStaging =
   '7FsWGsFnmVvRfMDpzz95Np76wf/1sPaK0Og9yiB+P8QbjiC8FV67NBans9hzZEkBaQMhiapzgMR6CkZIZPvgwQboAxl65JWRZecGe5V3XO4sdKeNemdAZ2TzQuWkuZoA'
 
 const signMoonpayUrlStaging = 'https://us-central1-celo-org-mobile.cloudfunctions.net/signMoonpay'
```

Comment:
function name for staging should be changed to `signMoonpayStaging` too.

Reply:
Thanks @jeanregisser ! I had left `signMoonpay` deployed to be compatible with existing clients (although it isn't being used), but thinking about this more it's important to have whatever is deployed match the code.